spk/shairport/src/dsm-control.sh

```diff
@@ -0,0 +1,84 @@
+#!/bin/sh
+
+# Package
+PACKAGE="shairport"
+DNAME="ShairPort"
+
+# Others
+SHAIRPORT="${SYNOPKG_PKGDEST}/bin/shairport"
+VAR_DIR="${SYNOPKG_PKGDEST}/var"
+PID_FILE="${VAR_DIR}/${PACKAGE}.pid"
+
+start_daemon ()
+{
+    # Launch the service in the background.
+    ${SHAIRPORT} --daemon --pidfile="$PID_FILE" --log=${VAR_DIR}/${PACKAGE}.log
```

Comment:
As Diaoul suggested, maybe use the start-stop daemon Busybox provides. See https://github.com/SynoCommunity/spksrc/blob/master/spk/tt-rss/Makefile#L33-L34.
Usage would look quite similar to this: https://github.com/SynoCommunity/spksrc/blob/master/spk/tt-rss/src/dsm-control.sh#L19.